### ISSUE TITLE
Support rounding in spatial decomposition elevation field

### ISSUE THREAD

**Issue by manuvarkey** [IfcOpenShell/IfcOpenShell#5461]
![image](https://github.com/user-attachments/assets/d41c4c54-71d1-4337-95c8-9ba031465389)

Decimal values entered in elevation field are not rounded and displays large number of decimal places.

**Comment by manuvarkey** [comment]
@Moult (as per discussion on matrix)

Is this the way to address this ? or what is the best approach ?

https://github.com/IfcOpenShell/IfcOpenShell/commit/2fdf97477c230db308e15c5fdb31a9952ed9ebb5

**Comment by Moult** [comment]
I don't think this is the right approach. I think we should implement a new function in util.unit perhaps which rounds to the precision value already defined in the IFC - of which the purpose is to show the precision of this type of length value (I think, ping @aothms )

**Comment by manuvarkey** [comment]
@Moult Where is precision defined in IFC ?

**Comment by Moult** [comment]
In the Precision attribute of IfcGeometricRepresentationContext - it'll be the Model context since it's a 3D coordinate.

**Comment by theoryshaw** [comment]
What if you have 5 different people working on the same IFC file and they want to change the tolerance of this read out in the ui?

**Comment by Moult** [comment]
Yes, we need both, but we need precision first, and then user-configurable "display" later on.

**Comment by manuvarkey** [comment]
@Moult Is the following implementation ok ?

https://github.com/IfcOpenShell/IfcOpenShell/pull/5532

**Comment by falken10vdl** [comment]
@manuvarkey since your proposal is merged I believe this can be closed.
Please tell us if otherwise. Thanks!
